app/Http/Requests/Publication/StorePublicationRequest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace App\Http\Requests\Publication;
+
+use App\Types\CitationIndex;
+use App\Types\PublicationType;
+use Carbon\Carbon;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class StorePublicationRequest extends FormRequest
+{
+    /**
+     * Prepare data for validation
+     *
+     * @return void
+     */
+    protected function prepareForValidation()
+    {
+        $this->merge([
+            'date' => $this['date']['month'] . ' ' . $this['date']['year'],
+            'is_published' => $this->filled('is_published'),
+        ]);
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = [
+            'type' => ['required', Rule::in(PublicationType::values())],
+            'paper_title' => ['required', 'string', 'max:400'],
+            'document' => ['required', 'file', 'mimetypes:application/pdf,image/*', 'max:200'],
+
+            'is_published' => ['required', Rule::in([true, false])],
+
+            'co_authors' => ['nullable', 'array', 'max:10', 'min:1'],
+            'co_authors.*.name' => ['required', 'string'],
+            'co_authors.*.noc' => ['required', 'file', 'max:200', 'mimeTypes:application/pdf, image/*'],
+
+            'name' => ['exclude_if:is_published,' . false, 'required', 'string', 'max:400'],
```

Comment:
I think you dont need the concatenation operator here... You can put false within the string as well